src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -898,6 +898,29 @@ void TLayout::layoutArticulation(const Articulation* item, Articulation::LayoutD
     ldata->setBbox(bbox.translated(-0.5 * bbox.width(), 0.0));
 }
 
+void TLayout::fillArticulationShape(const Articulation* item, Articulation::LayoutData* ldata)
```

Comment:
Isn't it possible to call this in `TLayout::layoutArticulation` instead of `EngravingItem::LayoutData::shape`?

Reply:
Good point 👍